README.md

```diff
@@ -229,6 +229,9 @@ The available options are the following (also documented in [hexdocs](https://he
     * `:smallest` - the smallest best fit unit will be used
     * `:none`     - no unit scaling will occur.
 * `:before_scenario`/`after_scenario`/`before_each`/`after_each` - read up on them in the [hooks section](#hooks-setup-teardown-etc)
+* `profile_after` - accepts any of the following options:
+      * a boolean   - `true` will enable profiling with the default profiler (`:cprof`) and `false` will disable profiling. Defaults to `false`.
+      * a profiler  - either as a tuple of `{profiler, opts}` (e.g., `{:fprof, [sort: :own]}`) or just the profiler, which is equivalent to `{profiler, []}`.
```

Comment:
a link on where to find what valid profilers are would be nice (i.e. hexdocs or just listing them all or both, whatever feels best for you!)

Reply:
another "e.g. `:fprof`" for just the formatter might be nice (I didn't understand this on first reading and only found out in the type spec :) )